zulip_bots/zulip_bots/bots/bugzilla/doc.md

```diff
@@ -0,0 +1,30 @@
+# Bugzilla bot
+
+This bot allows to update directly Bugzilla from Zulip
+
+## Setup
+
+To use Bugzilla Bot, first set up `bugzilla.conf`. `bugzilla.conf` takes 2 options:
+ - site (the site like `https://bugs.xxx.net` that includes both the protocol and the domain)
+ - api_key (a Bugzilla API key)
```

Comment:
I think it'd be best to have an example in a markdown code block; without that, it isn't clear what format this file should be in.

Reply:
I thought it was not required because an example was provided in the source code. But sure if it helps, it is worth adding it.
It is done in the last commit.